lib/components/map/default-map.tsx

```diff
@@ -383,8 +403,10 @@ const mapDispatchToProps = {
   bikeRentalQuery,
   carRentalQuery,
   setLocation,
+  setMapCenter,
   setMapPopupLocation,
   setMapPopupLocationAndGeocode,
+  setMapZoom,
```

Comment:
Remove unused prop.

Reply:
Removed in d3575aa5c7d8ea8b45d8959c9d74f64ac03b0274